### PR TITLE
Nerfs beepskys by reducing significantly the amount of time needed to break out of its zip ties

### PR DESCRIPTION
**Pull request by ClownMoff**
## About The Pull Request

Beepsky now uses a cheap ziptie that takes only 15 seconds to break from.

## Why It's Good For The Game

As it is, Beepsky is a do nothing -> win robot that can hunt antags and criminals alike and gives them in a plate to security while they do nothing. Getting the restrains off takes 45 seconds, and while that time is passing you can't  hide or open doors, so you are at the mercy of security. Instead of removing the ability to arrest people and handcuff them, this PR reduces the amount of time needed to break free to 15 seconds. Security still has some time to get to you, since Beepsky takes a couple seconds to get to you and arrest you and you can go almost anywhere in most stations from brig in that time. Since beepsky zipties are a special type of zipties only found in beepsky (which reduces a LOT the already low amount of avaiable zipties on the station), knowledge heretics now have to get regular handcuffs, which are a hard item to get anyways

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://cdn.discordapp.com/attachments/1141055730812727356/1218892032278265947/beepskyties2.mp4?ex=660950aa&is=65f6dbaa&hm=5c569c2b0ab8351303bfc6014bfd0b9204a7303259c9a2e6581898f6e78f0496&

Thanks to Aramix for the video
</details>

## Changelog
:cl: ClownMoff
balance: beepsky now uses a special type of zipties that takes 15 seconds to break from
tweak: Heretics with Ritual of Knowledge now get handcuffs instead of zipties as one of the possible items necessary.
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
Why does this touch heretics?

**Comment by DarnTheMarn** [comment]
i ded moment, but a valid change. They're a lot less bad now that they're not a one hit KO. I wonder if they'll get changed to be a 3 hit down now that batons are looking to be a 3 hit,

**Comment by zeskorion** [comment]
do they still have a 'detain' mode?

**Comment by ClownMoff** [comment]
Yes

File: code/game/objects/items/handcuffs.dm [hunk 1]
```diff
@@ -197,6 +197,22 @@
...
+	breakouttime = 150
+	trashtype = /obj/item/restraints/handcuffs/cable/zipties/beepsky/used
+
+/obj/item/restraints/handcuffs/cable/zipties/beepsky/used
+	desc = "A pair of broken zipties."
+	icon_state = "cuff_used"
+	item_state = "cuff"
+
+/obj/item/restraints/handcuffs/cable/zipties/beepsky/used/attack()
+	return
```
Comment (PowerfulBacon) [inline code/game/objects/items/handcuffs.dm]: whats the point of this if its exactly the same as the other type of zipties anyway?

**Review by PowerfulBacon — COMMENTED** [review]
I mean this seems fine, since the breakout time of 15 seconds is still insanely long anyway. You eitiher undo the cuffs and people catch up to you, or you run away and get chased and have no time to undo cuffs.

**Comment by zeskorion** [comment]
> Yes

its sooo much more bs than the cuffs are

File: code/game/objects/items/handcuffs.dm (hunk 1 above)
Comment (ClownMoff) [reply code/game/objects/items/handcuffs.dm]: Makes sense. I deleted it.

**Review by PowerfulBacon — APPROVED** [review]

File: code/modules/mob/living/simple_animal/bot/secbot.dm [hunk 2]
```diff
@@ -234,7 +234,7 @@
 	if( !on || !Adjacent(C) || !isturf(C.loc) ) //if he's in a closet or not adjacent, we cancel cuffing.
 		return
 	if(!C.handcuffed)
-		C.handcuffed = new /obj/item/restraints/handcuffs/cable/zipties/used(C)
+		C.handcuffed = new /obj/item/restraints/handcuffs/cable/zipties/beepsky/used(C)
```
Comment (PowerfulBacon) [inline code/modules/mob/living/simple_animal/bot/secbot.dm:237]:
```suggestion
		C.handcuffed = new /obj/item/restraints/handcuffs/cable/zipties/beepsky(C)
```

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

File: code/game/objects/items/handcuffs.dm [hunk 3]
```diff
@@ -197,6 +197,14 @@
 /obj/item/restraints/handcuffs/cable/zipties/used/attack()
 	return
 
+/obj/item/restraints/handcuffs/cable/zipties/beepsky
```
Comment (PowerfulBacon) [inline code/game/objects/items/handcuffs.dm:200]: Previous review was incorrect, sorry. The cuffs being applied by beepsky are the used type which means that this typepath is pointless but the used typepath needs to be the one that exists instead.

The typepath should just be `/obj/item/restraints/handcuffs/cable/zipties/used` with a breakouttime set and no other variables set.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Inco

**Pull request closed without merging**